Fatal error: exception Failure("expecting 2 arguments in Call(foo, IntLit(42), BoolLit(True), BoolLit(False))")
